src/TSTransformer/nodes/expressions/transformTemplateExpression.ts

```diff
@@ -33,23 +32,17 @@ export function transformTemplateExpression(state: TransformState, node: ts.Temp
 
 	for (let i = 0; i < node.templateSpans.length; i++) {
 		const templateSpan = node.templateSpans[i];
-		let expression = orderedExpressions[i];
+		const expression = orderedExpressions[i];
 		const type = state.getType(templateSpan.expression);
-		if (!isDefinitelyType(type, isStringType)) {
-			if (isDefinitelyType(type, isLuaTupleType(state))) {
-				DiagnosticService.addDiagnostic(errors.noLuaTupleInTemplateExpression(templateSpan.expression));
-			}
-			if (ts.isCallExpression(templateSpan.expression) && isPossiblyType(type, isUndefinedType)) {
-				expression = luau.create(luau.SyntaxKind.ParenthesizedExpression, { expression });
-			}
-			expression = luau.call(luau.globals.tostring, [expression]);
+		if (isDefinitelyType(type, isLuaTupleType(state))) {
+			DiagnosticService.addDiagnostic(errors.noLuaTupleInTemplateExpression(templateSpan.expression));
```

Comment:
Do we need to keep this diagnostic? Putting a LuaTuple in an InterpolatedString should wrap it, so it's not much different from putting another type of table in. Unusual, probably unintentional, but not wrong. More something for a lint rule than a compiler error.